model/src/main/java/org/mskcc/cmo/metadb/model/SampleManifestEntity.java

```diff
@@ -470,12 +471,12 @@ public void setSampleStatus(String sampleStatus) {
         this.sampleStatus = sampleStatus;
     }
 
-    public Date getCreationDate() {
-        return creationDate;
+    public Date getLimsCreationDate() {
+        return limsCreationDate;
     }
 
-    public void setCreationDate(Date creationDate) {
-        this.creationDate = creationDate;
+    public void setLimsCreationDate(Date limsCreationDate) {
+        this.limsCreationDate = limsCreationDate;
```

Comment:
I still dont see the lims creation date in the source json? where is this from?

Reply:
It came with lims data members for SampleManifest. The json published by lims doesn't have it so the value of limsCreationDate is set to null. Null data members are not added to the json.

Reply:
okay let's remove this then

Reply:
can you maybe link where in their code this is?

Reply:
I think we can remove ti since lims isn't sending this information in their json

Reply:
oh that might have been something that _we_ introduced for our own graph db. Yes let's definitely remove  it then